<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordWindow</class>
 <widget class="QMainWindow" name="WordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Word</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 238, 176);
/*background-color: rgb(39, 80, 78);*/</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="EnglishWord">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>801</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 72pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;One&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LeftMove">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RightMove">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
   border: 2px solid grey;
   border-radius: 5px;
   background-color: #FFFFFF;
}

QProgressBar::chunk {
   background-color: #05B8CC;
   width: 20px;
}

QProgressBar {
   border: 2px solid grey;
   border-radius: 5px;
   text-align: center;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="InitialPicture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterButton">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>370</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Bodoni MT&quot;;
</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QToolButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/addPicture.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/deletePicture.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="changeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/changePicture.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="EmptyListPicture">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/EmptyList.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="EmptyListLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string>The list is empty</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ChineseMeaning">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <item>
     <property name="text">
      <string>Row 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Row 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Row 3</string>
     </property>
    </item>
   </widget>
   <zorder>EnglishWord</zorder>
   <zorder>LeftMove</zorder>
   <zorder>RightMove</zorder>
   <zorder>InitialPicture</zorder>
   <zorder>progressBar</zorder>
   <zorder>EnterButton</zorder>
   <zorder>addButton</zorder>
   <zorder>deleteButton</zorder>
   <zorder>changeButton</zorder>
   <zorder>EmptyListPicture</zorder>
   <zorder>EmptyListLabel</zorder>
   <zorder>ChineseMeaning</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
